### DISCUSSION TITLE
Quick Launch toolbar.

### DISCUSSION THREAD

**Discussion by Snik99** [valinet/ExplorerPatcher#3608] [Q&A] [answered]
Hi all. Just installed win11 ltsc iot and I really miss my quick launch tool bar. Is this something being worked on here, or am I barking up the wrong tree..??
Thanks.

**Comment by pyrates999** [comment, marked as answer]
Install the latest pre-release of EP: https://github.com/valinet/ExplorerPatcher/releases

Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

Then check this link for how to create it: https://eileenslounge.com/viewtopic.php?style=34&p=316753

It will guide you on how to create a new toolbar that points to a directory with shortcuts in it.

**Reply by Snik99** [reply]
Hey thanks man, downloaded and installed. Just working now on setting it up to my liking, on the left hand side.
Thanks again.

**Reply by pyrates999** [reply]
Please set my response as the answer then.

**Comment by Snik99** [comment]
One little thing,I used to drag it to the left hand side(I know, I'm a freak) but my  method doesn't work nice under this new system. Is there a way, or asking too much..??

Thanks.

**Reply by pyrates999** [reply]
Right click taskbar and untick lock the taskbar.  Then you should be able to move it.

**Reply by Snik99** [reply]
Thanks. Yea, it does indeed do that. However it forces open icons over to about a 1 inch space on the right hand side of the taskbar, rendering it unusable.

**Reply by pyrates999** [reply]
Please show a screen shot of what you mean.

**Reply by Snik99** [reply]
[img]https://i.postimg.cc/pLBJc90k/QL.png[/img][/url]
You see on the left, the icons for items currently in the quicklaunch. On the right, the scrunched up icons of open items.

The text in the image is not related to the  issue.

**Reply by pyrates999** [reply]
Please wait for a developer to respond.

**Reply by Snik99** [reply]
Thanks mate, really appreciate all the help. You rock...!!
